### PR TITLE
Additional changes to the create command, to produce valid builder image contents.

### PR DESCRIPTION
**Pull request by soltysh**
@mfojtik ptal

### CODE REVIEW

File: pkg/create/templates/test.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ container_ip() {
 }
 
 run_sti_build() {
-  sti build ${sti_args} file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
+  sti build ${sti_args} ${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
```
Comment (mfojtik) [inline pkg/create/templates/test.go]: add file:/// back, it is the preferred way. The other one will give you fat warning.

Comment (rhcarvalho) [inline pkg/create/templates/test.go]: Hey, I see there's no more `git init` step in `prepare()` ... that means that actually using `file://` produce a bad test runner, one that won't work

Comment (mfojtik) [inline pkg/create/templates/test.go]: we should use --context-dir in that case...

**Comment by soltysh** [comment]
@mfojtik reverted the git init step in `test/run`.

**Comment by mfojtik** [comment]
LGTM

**Comment by soltysh** [comment]
[merge]

**Pull request merged**